client/my-sites/email/email-different-domain-owner-message/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslate } from 'i18n-calypso';
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+type EmailDifferentDomainOwnerMessageProps = {
+	source: string;
+};
+
+export const EmailDifferentDomainOwnerMessage = (
+	props: EmailDifferentDomainOwnerMessageProps
+) => {
+	const { source } = props;
```

Comment:
Given that this type is only used once and that it only defines one parameter, I think we could just inline it:
```suggestion
export const EmailDifferentDomainOwnerMessage = (
	{ source: string }
) => {
```